content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md

```diff
@@ -22,15 +22,15 @@ $ helm template $HOME/istio-fetch/istio \
   --set global.k8sIngress.enabled=true \
   --set global.k8sIngress.enableHttps=true \
   --set global.k8sIngress.gatewayName=ingressgateway \
-  --set certmanager.enabled=true \
-  --set certmanager.email=mailbox@donotuseexample.com \
   > $HOME/istio-fetch/istio.yaml
 {{< /text >}}
```

Comment:
Also, lines 18-26 need to be indented 4 spaces for the numbered list to work.

Reply:
Thanks! verified with `make serve` this works now